Makefile

```diff
@@ -170,7 +178,7 @@ quiet = $(if $(V),$($(1)),$(call output,$1,$@);$($(1)))
 
 lib: $(libceed) $(ceed.pc)
 # run 'lib' target in parallel
-all:;@$(MAKE) $(MFLAGS) V=$(V) lib
+par:;@$(MAKE) $(MFLAGS) V=$(V) lib
```

Comment:
Now phony target? Can we delete or make into a meaningful name like `lib-parallel`?

Reply:
I wanted to use the target `all` for building the library with all the examples and all the tests - I think that is a more appropriate use of that target.

That is why, I just renamed the target to something short `all` -> `par` -- I think @camierjs was the main user of this target and I assumed he'd prefer something short to type. I guess I forgot to make it a phony target -- I'll do that in a moment.

Reply:
I'm not a fan of Makefile cleverness/abbreviations, but if someone is still attached to this, I won't argue.